rest/route/github.go

```diff
@@ -302,7 +291,7 @@ func (gh *githubHookApi) handleCheckRunRerequested(ctx context.Context, event *g
 			"repo":    repo,
 			"message": "check run doesn't carry task",
 		})
-		return gimlet.NewJSONInternalErrorResponse(errors.New("check run doesn't carry task"))
+		catcher.Add(errors.New("check run doesn't carry task"))
```

Comment:
Why are we continuing to `taskToRestart, taskErr := data.FindTask(checkRunTask)` when the checkRunTask == ""

Reply:
we should error good catch!